akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/MaxThroughputSpec.scala

```diff
@@ -174,7 +197,8 @@ object MaxThroughputSpec extends MultiNodeConfig {
       case EndResult(totalReceived) ⇒
         val took = NANOSECONDS.toMillis(System.nanoTime - startTime)
         val throughput = (totalReceived * 1000.0 / took)
-        println(
+
+        reporter.reportResults(
           s"=== MaxThroughput ${self.path.name}: " +
             f"throughput ${throughput * testSettings.senderReceiverPairs}%,.0f msg/s, " +
             f"${throughput * payloadSize * testSettings.senderReceiverPairs}%,.0f bytes/s (payload), " +
```

Comment:
This turns out quite hard to fix. The problem is that now that I changed to n-to-n communication, all the phases of the test, warmup, running and test reporting need to be synchronized over all the actors which is tricky to get right...